src/AI/Muriel.java

```diff
@@ -495,7 +502,7 @@ private boolean queueUnitAction(GameMap gameMap, Unit unit, boolean allowDeferri
 
     //////////////////////////////////////////////////////////////////
     // See if there's something to capture (but only if we are moderately healthy).
-    if( unit.getHealth() > 7 )
+    if( unit.getHealth() >= UnitHealthThresholds.ACTION_READY )
```

Comment:
Technically a change in logic, but consistent with line 560 (and my remembered intent that the unit can start a capture if it it would take three or fewer turns to complete).